### ISSUE TITLE
When will the training code be open source?

### ISSUE THREAD

**Issue by fuchao01** [Atlas200dk/sample-imageinpainting-HiFill#14]
When will the training code be open source?

**Comment by Pinakee15** [comment]
Yes how long sir?

**Comment by duxingren14** [comment]
https://github.com/duxingren14/Hifill-tensorflow

**Comment by Pinakee15** [comment]
Hello @duxingren14 <https://github.com/duxingren14> , first of all thanks
you very much for your this wonderful contribution on image inpainting.
I have a doubts regarding the training of files -
Which images to be put in the train folder . Whether to put the plain real
raw image or the one with having masked region on it to be removed and
similarly which images to put in the "val" folder? COuld you educate me on
thsi thank you and regards..!
